services/kube-prometheus-stack/70.4.2/defaults/cm.yaml

```diff
@@ -341,6 +341,8 @@ data:
       enabled: true
       defaultDashboardsEnabled: true
       priorityClassName: "dkp-critical-priority"
+      persistence:
+        enabled: true
```

Comment:
Enabling this runs an additiona init pod for adjusting the data ownership. Can we add values for using existing `kubectl` image - https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml#L449-L463 

Otherwise we'd have to bring another busybox image to the airgapped bundle.